docs/installation/windows.md

```diff
@@ -147,10 +172,25 @@ Forem requires a version of Elasticsearch between 7.1 and 7.5. Version 7.6 is
 not supported. We recommend version 7.5.2.
 
 We recommend following the install guide
-[in Elasticsearch's docs](https://www.elastic.co/guide/en/elasticsearch/reference/7.5/zip-windows.html)
+[in Elasticsearch's docs](https://www.elastic.co/guide/en/elasticsearch/reference/7.5/targz.html#install-linux)
 for installing on Windows machines.
 
-NOTE: Make sure to download **the OSS version**, `elasticsearch-oss`.
+_NOTE:_ Make sure to download **the OSS version**, `elasticsearch-oss` by
+following the following steps:
+
+1. Replace the link in the first `wget` command with the Apache 2.0 licensed
+   code link listed below the code-block.
+2. For the second `wget` command add `.sha512` to the end of the Apache 2.0
+   licensed code link in order to download the correct checksum file.
+3. For the `shasum` command, replace the file name with the new file downloaded
+   by the second wget command.
```

Comment:
Wouldn't it be easier to just do these steps ourselves and put the instructions in a code block?
Like we do in the macOS version: https://docs.dev.to/installation/mac/ section "Installing Elasticsearch from the archive"

Reply:
Yeah, I was in two minds about that while writing this up. I think since `7.5.2` isn't the latest anymore, it should be safe to switch to preset commands. I will make those changes!

Reply:
Do you think a similar approach for installing WSL 2 would be preferable too?

Reply:
@Amorpheuz no I think they can figure that out :D unless you think the original documentation is lacking on that